packages/css/src/components/select/select.scss

```diff
@@ -34,33 +34,32 @@
   }
 }
 
-.ams-select:invalid,
-.ams-select[aria-invalid="true"] {
-  border-color: var(--ams-select-invalid-border-color);
-}
-
 .ams-select:disabled {
   color: var(--ams-select-disabled-color);
   cursor: var(--ams-select-disabled-cursor);
+}
 
-  &:not([multiple]) {
-    background-image: var(--ams-select-disabled-background-image);
-
-    /* stylelint-disable no-descending-specificity */
-    &:hover {
-      background-image: var(--ams-select-hover-background-image);
-    }
-  }
+.ams-select:disabled:not([multiple]) {
```

Comment:
The other two occurrences of `:not([multiple])` in this file use a nested parent selector, this one doesn’t. Let’s pick one approach.